packages/frontend-api/src/Model/Resolver/Advert/AdvertsQuery.php

```diff
@@ -44,7 +48,12 @@ public function advertsQuery(?string $positionName = null, ?string $categoryUuid
         $category = null;
 
         if ($categoryUuid !== null) {
-            $category = $this->categoryFacade->getByUuid($categoryUuid);
+            try {
+                $category = $this->categoryFacade->getByUuid($categoryUuid);
+            } catch (CategoryNotFoundException $exception) {
+                $seoMixCategory = $this->readyCategorySeoMixFacade->getByUuid($categoryUuid);
+                $category = $this->categoryFacade->getById($seoMixCategory->getCategory()->getId());
```

Comment:
Isn't this sufficient?

```suggestion
                $category = $seoMixCategory->getCategory();
```

Reply:
fixed 😅